server.js

```diff
@@ -0,0 +1,40 @@
+// Server
+var express = require('express');
+var app = express();
+var path = require('path');
+var server = require('http').createServer(app);
+var io = require('socket.io')(server);
+var hipsterList = require('./public/js/hipsterList');
+var bodyParser = require('body-parser');
+
+//data
+var port = process.env.PORT || 3000;
+var db = new hipsterList();
```

Comment:
I think this is misleading to call this a db, I think a more descriptive name would make it clearer what it is - maybe `Items`? With a variable name `items`?